include/robot_localization/ros_filter.h

```diff
@@ -103,6 +105,28 @@ struct CallbackData
   bool relative_;
   double rejectionThreshold_;
   double rejectionThresholdInit_;
+  double rejectionThresholdTrusted_;
+};
+
+struct SourceData
+{
+  SourceData()
+  {
+    bias_pose_.resize(STATE_SIZE);
+    bias_pose_.setZero();
+    bias_twist_.resize(STATE_SIZE);
+    bias_twist_.setZero();
+    bias_acceleration_.resize(STATE_SIZE);
+    bias_acceleration_.setZero();
+  }
+
+  bool trusted_{false};
+  double last_trusted_s_{-99999.0};  // Never trusted initially
+  Eigen::VectorXd bias_pose_;
+  Eigen::VectorXd bias_twist_;
+  Eigen::VectorXd bias_acceleration_;
+  bool bias_valid_{false};  // Invalid initially
+  double last_bias_s_{-99999.0};  // Never set initially
```

Comment:
Is there any reason to not use `-std::numeric_limits<double>::infinity()` instead of `-99999.0`?

Reply:
No, no reason. I'll change it.